### ISSUE TITLE
Question: Run multiple instances of eZ Launchpad

### ISSUE THREAD

**Issue by SalvatorePollaci** [ezsystems/launchpad#34]
Hi,
is it possible to run multiple instances of eZ Launchpad. I.e. run projects on different ports?

**Comment by raupie** [comment]
Yep! In .ezlaunchpad.yml file there is a port variable. Also the wizard prompts you for a port number. I do this for setting up sandboxes for ezplatform without messing with another project.

**Comment by SalvatorePollaci** [comment]
@raupie Thank you very much!

**Comment by Plopix** [comment]
Thanks @raupie and @SalvatorePollaci for the question ;)
